### PR TITLE
38 use parcours

### PR DESCRIPTION
**Pull request by llaulier**

### CODE REVIEW

**Review by RoxaneDrelon — DISMISSED** [review]
SnE

File: src/components/marker/MarkerFullParcours.jsx [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+  const handleOpen = () => {
+    setOpen(true);
+  };
+
+  const handleClose = () => {
+    setOpen(false);
+  };
+  const uniqueCapsuleById = _.uniqBy(allCaps, 'capsule_id');
+
+  console.log(id);
```
Comment (LoicBrassart) [inline src/components/marker/MarkerFullParcours.jsx]: Nope!

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
console.log to delete, then notify me and it'll pass!

File: src/components/marker/MarkerFullParcours.jsx (hunk 1 above)
Comment (RoxaneDrelon) [reply src/components/marker/MarkerFullParcours.jsx]: I specifically search for them though ...

**Review by RoxaneDrelon — APPROVED** [review]
SnE

**Review by LoicBrassart — APPROVED** [review]
SnE

**Pull request merged**